pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -175,6 +175,9 @@ func (c *schedulingController) sync(ctx context.Context, syncCtx factory.SyncCon
 	}
 
 	// update placement status if necessary
+	if clusters == nil {
+		scheduleResult.unscheduled = -1
```

Comment:
The clusters is nil when
- No clustersetbinding found in the placement namespace;
- Or the clustersets bound to the placement namespace have no intersection with clustersets specified in .spec.clusterSets;

So in this case, we may not be able to set the condition with reason NoManagedClusterSetBindings.

Reply:
I guess I can put the check deeper, where we pull the ManagedClusterSetBindings